### ISSUE TITLE
Unable to set to the ValidFrom of the latest version

### ISSUE THREAD

**Issue by akiqsinco** [sergeyshushlyapin/Sitecore.FakeDb#200]
I want to set `Item.Publishing.ValidFrom` property of the latest version.
The code is below.

```csharp
using (var db = new Db
{
    // create versioned item
    new DbItem("Home")
    {
        new DbField("Text")
        {
            { "en", 1, "" },
            { "en", 2, "" },
        }
    }
})
{
    // get version 2
    var homeV2 = db.GetItem("/sitecore/content/Home", "en", 2);

    // set ValidFrom and save.
    using (new EditContext(homeV2))
    {
        homeV2.Publishing.ValidFrom = new DateTime(2018, 10, 25, 15, 14, 00, DateTimeKind.Utc);
    }

    // reget version 2
    homeV2 = db.GetItem("/sitecore/content/Home", "en", 2);
    var validFromV2 = homeV2.Publishing.ValidFrom;  // => 0001/01/01 00:00:00

    // get version 1
    var homeV1 = db.GetItem("/sitecore/content/Home", "en", 1);
    var validFromV1 = homeV1.Publishing.ValidFrom;  // => 2018/10/25 15:14:00
}
```

The expected value of version 2 is `2018/10/25 15:14:00`, but the value is `DateTime.MinValue` (not set).
And the value of version 1 is set to `2018/10/25 15:14:00`.

Thank you.

**Comment by sergeyshushlyapin** [comment]
Thanks for reporting this, @akiqsinco.

That's definitely a bug. The expected result is the following:
- v1 is `DateTime.MinValue`
- v2 contains the date specified

**Comment by sergeyshushlyapin** [comment]
Released as [v1.7.3](https://github.com/sergeyshushlyapin/Sitecore.FakeDb/releases/tag/v1.7.3).
